src/deploy-tes-on-azure/deploy-tes-on-azure.csproj

```diff
@@ -24,7 +24,7 @@
   <ItemGroup>
     <PackageReference Include="Azure.Identity" Version="1.8.0" />
     <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.4.0" />
-    <PackageReference Include="KubernetesClient" Version="9.1.3" />
+    <PackageReference Include="KubernetesClient" Version="10.0.1" />
```

Comment:
Are the changes in this file necessary?  If not, let's not make changes that aren't directly related to supporting the sovereign cloud

Reply:
Downgrade to 9.1.3